keps/sig-cluster-lifecycle/kubeadm/20190722-Advanced-configurations-with-kubeadm-(Kustomize).md

```diff
@@ -185,7 +178,7 @@ patches and/or patches for a subset of nodes.
 
 The resulting workflow for using in kustomize will be the following
 
-1. Create a folder with some patches, e.g.
+1.. Create a folder with some patches, e.g.
```

Comment:
is the extra dot required?

Reply:
I replaced `..` with `-` to make numbering less weird